app/livechat/client/views/app/tabbar/visitorEdit.js

```diff
@@ -81,9 +81,8 @@ Template.visitorEdit.onCreated(function() {
 
 	const rid = Template.currentData().roomId;
 
-	this.subscribe('livechat:rooms', { _id: rid });
-	this.autorun(() => {
-		const room = LivechatRoom.findOne({ _id: rid });
+	this.autorun(async () => {
+		const { room } = await APIClient.v1.get(`livechat/rooms/${ rid }`);
```

Comment:
What happens if the `APIClient` call fails? 
Wouldn't it be good to handle an eventual fail?

If not, just forget this comment..